db/seedfiles/translations/insured.rb

```diff
@@ -0,0 +1,395 @@
+# Insured Translations
```

Comment:
I don't think you should store keys that aren't explicit to the same namespace in the same files.

you have en.insured.consumer_roles etc but also ones that are general keys. If they're truly generic they should be in a shared file. 

@cdeyoung  can you comment for how that should be handled